tox.ini

```diff
@@ -2,9 +2,14 @@
 ; under all supported Python interpreters
 
 [tox]
-envlist = py27,py33,py34,py35
+envlist = py27,py34,py35,py36
 
 [testenv]
+basepython =
+    py27: python2.7
+    py34: python3.4
+    py35: python3.5
+    py36: python3.6
```

Comment:
is this needed? I was thinking it is automatic

Reply:
Yeah, I added it when debugging the python versions problems.